types/mui-datatables/index.d.ts

```diff
@@ -230,7 +230,7 @@ export interface MUIDataTableOptions {
     onFilterDialogOpen?: () => void;
     onFilterDialogClose?: () => void;
     onRowClick?: (rowData: string[], rowMeta: { dataIndex: number; rowIndex: number }) => void;
-    onRowsDelete?: (rowsDeleted: any[]) => void;
+    onRowsDelete?: (rowsDeleted: {lookup: {[dataIndex: number]: boolean}, data: Array<{index: number, dataIndex: number}>}) => void;
```

Comment:
```suggestion
onRowsDelete?: (rowsDeleted: {
        lookup: { [dataIndex: number]: boolean };
        data: Array<{ index: number; dataIndex: number }>;
    }) => void;
```

Applied some autoformatter to make this much more readable